bindings/java/src/main/java/org/apache/opendal/BlockingOperator.java

```diff
@@ -37,8 +39,15 @@ public class BlockingOperator extends NativeObject {
      * @param schema the name of the underneath service to access data from.
      * @param map a map of properties to construct the underneath operator.
      */
-    public BlockingOperator(String schema, Map<String, String> map) {
-        super(constructor(schema, map));
+    public static BlockingOperator of(String schema, Map<String, String> map) {
```

Comment:
This is somehow a breaking change. But we're in the 0.x version series and this change free the users from limitation of constructors (must call super first, must initialize, etc.)

Reply:
I'm glad to add some notes if we have a proper place to write CHANGELOGS or Release Note.

Reply:
We can give the binding its changelog if needed. How do you feel about adding it as a separate file? Like our rust core.

Reply:
> Like our rust core

May you share a link?

Reply:
https://github.com/apache/incubator-opendal/blob/main/core/src/docs/upgrade.md

More like an upgrade guide. 🤔

Reply:
Updated.